packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div
+    id="sfHeader"
```

Comment:
@justyna-13 maybe you can use height by variable, check isMobile watcher 👍

Reply:
![Zrzut ekranu 2020-05-7 o 10 28 45](https://user-images.githubusercontent.com/12138170/81272267-8f57d100-904d-11ea-8bbf-8921fd4975c7.png)
maybe you can set this.height as Number and add  computed holderHeight(){} to return {height: `${this.height}px`}